reconcile/test/external_resources/test_rds_timeouts.py

```diff
@@ -0,0 +1,39 @@
+from unittest.mock import Mock
+
+import pytest
+
+from reconcile.external_resources.aws import AWSRdsFactory
+from reconcile.external_resources.model import (
+    ExternalResource,
+    ExternalResourceModuleConfiguration,
+    ExternalResourceProvision,
+)
+
+
+def test_timeout_values() -> None:
+    f = AWSRdsFactory(er_inventory=Mock(), secret_reader=Mock())
+
+    assert f._get_timeout_value("60m") == 60
+    assert f._get_timeout_value("1h") == 60
+    with pytest.raises(
+        ValueError,
+        match=r".*Invalid RDS instance timeout format: 12mm. Specify timeout in minutes\(m\) or hours\(h\)..*",
+    ):
+        assert f._get_timeout_value("12mm")
+
+
+def test_validate_timeouts() -> None:
+    f = AWSRdsFactory(er_inventory=Mock(), secret_reader=Mock())
+    resource = ExternalResource(
+        data={"timeouts": {"create": "60m", "update": "60m", "delete": "60m"}},
+        provision=Mock(spec=ExternalResourceProvision),
+    )
+    module_conf = ExternalResourceModuleConfiguration(reconcile_timeout_minutes=120)
+    f._validate_timeouts(resource, module_conf)
+
+    module_conf = ExternalResourceModuleConfiguration(reconcile_timeout_minutes=40)
+    with pytest.raises(
+        ValueError,
+        match=r".*RDS instance create timeout value 60 exceeds the module reconcile_timeout_minutes value 40.*",
+    ):
+        f._validate_timeouts(resource, module_conf)
```

Comment:
don't test private method, make it public or test `f.validate` instead.

Reply:
Changed the test methods to use validate() instead